### DISCUSSION TITLE
triple a suggestion

### DISCUSSION THREAD

**Discussion by the-kings-shilling** [triplea-game/triplea#10927] [Feature Requests & Ideas]
I would like to suggest a version of triple a in which takes into account the non-aggression pact. there should be a version in which puts the USSR as a AXIS ally. I would also add either 2subs/1 cruiser to the Dutch east indies since the bulk of the Dutch fleet to include all their heavy units were in the DEI.

**Comment by RoiEXLab** [comment]
This repository is focused on the developement of the engine. If you have any ideas you might want to discuss them in [the forum](https://forums.triple-game.org) or you can read through the [map making docs](https://github.com/triplea-game/triplea/tree/master/docs/map-making) and consider making a map mod or an entirely new map based on the maps you can find here: https://github.com/triplea-maps/

**Reply by LouisDesyjr** [reply]
As long as we want/need to make the games some what histoircal, I would put it to people that having the USRR as a possible Axis ally as a highly unlikely possibility for a number of reasons. 1: While Stalin did not trust the western democracies and there even was a British bombing raid on Baku early in the war, He most certainly trusted the fascist powers even less; especially considering in Mein Kamf Hitler wrote about Germany needing living space, which could only come from Russia plus I think there are a number of passages in Mein Kamf where Hitler talks about how Communism was the enemy of Germany; and lets not forget it was only a little more than 20 years earlier that Germany conquored Russia in World War 1 an took large areas of territory in the peace treaty with it. (Counterblanance with all that was the Western Allied troops deployed into Russia in support of the White army during the civil war, even near the end there was a White Army advancing on St. Petersburg with naval gunfire support from British naval units.)

**Comment by the-kings-shilling** [comment]
All accurate and valid points.

I was just toying with the possibility that the non-aggression treaty which they both worked with that allowed them both to annex territory (Poland, The Baltics) could have been the foundation for a repeat with Germany and Russia dividing up the middle east and brokering s deal with Japan for parts of China.

Imagine the challenge of the allies trying to overcome those interior lines and resources.



“There is nothing more deceptive than an obvious fact.”-Arthur Conan Doyle, The Boscombe Valley Mystery

Cade Logue
________________________________
